app/src/main/java/com/example/habitsmasher/ui/dashboard/AddHabitDialog.java

```diff
@@ -94,7 +94,7 @@ public void onClick(View view) {
                 // if the habit is valid, add it to the local list and external db
                 if (habitValidator.isHabitValid(habitTitle,
                                                 habitReason,
-                                                habitDate, _tracker)){
+                                                habitDate, _tracker, _publicPrivateButtons)){
```

Comment:
minor: can we new line the `tracker` and `publicPrivateButtons` so they follow the style of the other parameters?